<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LAPTOP-BVMK5C57" failures="1" tests="1" name="Default test" time="7.122" errors="0" timestamp="2022-06-28T15:28:25 EEST">
  <testcase classname="Test.AllTest" name="UserLoginNeg" time="2.654">
    <failure type="java.lang.AssertionError" message="expected [Įvestas prisijungimo vardas ir/ arba slaptažodis yra neteisingi] but found [Ávestas prisijungimo vardas ir/ arba slaptaþodis yra neteisingi]">
      <![CDATA[java.lang.AssertionError: expected [Įvestas prisijungimo vardas ir/ arba slaptažodis yra neteisingi] but found [Ávestas prisijungimo vardas ir/ arba slaptaþodis yra neteisingi]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Test.AllTest.UserLoginNeg(AllTest.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- UserLoginNeg -->
  <testcase classname="Test.AllTest" name="UserLoginPoz" time="0.0">
    <ignored/>
  </testcase> <!-- UserLoginPoz -->
  <testcase classname="Test.AllTest" name="RegistrationPoz" time="0.0">
    <ignored/>
  </testcase> <!-- RegistrationPoz -->
  <testcase classname="Test.AllTest" name="UserLogOut" time="0.0">
    <ignored/>
  </testcase> <!-- UserLogOut -->
  <testcase classname="Test.AllTest" name="RegistrationNeg" time="0.0">
    <ignored/>
  </testcase> <!-- RegistrationNeg -->
</testsuite> <!-- Default test -->
